src/rgw/driver/sfs/sqlite/objects/object_definitions.h

```diff
@@ -43,8 +41,6 @@ struct DBOPObjectInfo {
   ceph::real_time mtime;
   ceph::real_time set_mtime;
   ceph::real_time delete_at;
-  rgw::sal::Attrs attrs;
```

Comment:
why are we removing the object attrs from the object?

Reply:
We are moving  `attrs` from `objects` to `versioned_objects`.
This because every version of an object is a logically separate entity from every other version.
Every version has its own attributes.
Non versioned objects have also a row inside `versioned_objects` (it can be intended as the only existing version).
moving `attrs` to `versioned_objects` uniforms the code for both `object` and `versioned-object` types.

Reply:
ack